CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [8.1.1]
+### Added
+* Support for a new water masking dataset based off of OpenStreetMaps and ESA WorldCover data.
+### Removed
+* Polygon processing functions: `split_geometry_on_antimeridian` and `get_envelope_wgs84` from `water_mask.py`.
```

Comment:
@forrestfwilliams Are these user-facing functions? If so, we should make this release v9.0.0. If not, they probably don't need to appear in the Changelog?